spec/requests/tasks_spec.rb

```diff
@@ -2,6 +2,17 @@
 
 RSpec.describe "Tasks", type: :request do
   describe "GET /index" do
-    pending "add some examples (or delete) #{__FILE__}"
+    it "gets a list of tasks" do
+      user = User.create(name: "Renita", email: "hello@world.com", password: "1234hi")
```

Comment:
Make sure that your user includes all attributes needed to create a user.  Down the line this test will fail when validations are added in.

Reply:
So, just to clarify - we already did do our validations. We are intentionally making only the name be required, not the profile picture or bio. Is there something I'm missing?